pkgs/servers/mail/mailman/default.nix

```diff
@@ -1,23 +1,43 @@
-{ lib, buildPythonPackage, fetchPypi, fetchpatch, isPy3k, alembic, aiosmtpd, dnspython
+{ lib, buildPythonPackage, fetchPypi, fetchpatch, isPy3k, python3, postfix, lynx
+, python-editor
+, aiosmtpd, dnspython
 , flufl_bounce, flufl_i18n, flufl_lock, lazr_config, lazr_delegates, passlib
 , requests, zope_configuration, click, falcon, importlib-resources
-, zope_component, lynx, postfix, authheaders, gunicorn
+, zope_component, authheaders, gunicorn
+# , alembic, sqlalchemy
 }:
 
+let
+  # Mailman does not support sqlalchemy >= 1.4 https://gitlab.com/mailman/mailman/-/issues/899
+  pythonOverride = python3.override {
+    packageOverrides = self: super: {
+      sqlalchemy = super.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
```

Comment:
This is unfortunate. I'm not sure what to suggest here though. I will have to defer to someone who might know what's the standard practice for these kinds of problems.

Reply:
This is alright to do and mailman is not the only package doing it.